manifests/t/TechSmith/Camtasia/23.4.5.52812/TechSmith.Camtasia.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TechSmith.Camtasia
+PackageVersion: 23.4.5.52812
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19042.0
+InstallerType: burn
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+    MinimumVersion: 92.0.902.78
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    MinimumVersion: 14.27.29112.0
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.techsmith.com/camtasiastudio/releases/2345/camtasia.exe
+  InstallerSha256: C19B1D608D6003C8F963D4170B7E84B7CBD4F4849B0E700EAD5D976A77A0158A

```

Comment:
```suggestion
  InstallerSha256: D652ACE87DC6B2B4BCF85E17E099A6928414134A71E4F5E45E580148C759A685
```

Reply:
@denelon / @Madhusudhan-MSFT - Do the pipelines not specify `--dependency-source winget` ?